makefile

```diff
@@ -291,6 +306,16 @@ else
 	@echo 'Info: Pylint requires Python 2; skipping this step on Python $(python_major_version)'
 endif
 
+flake8.log: makefile $(flake8_rc_file) $(flake8_py_files)
+	rm -f flake8.log
+	-bash -c "set -o pipefail; PYTHONPATH=. flake8 --config=$(flake8_rc_file) $(flake8_py_files) 2>&1 |tee flake8.tmp.log"
```

Comment:
Removing the minus from the bash command would mean we enable flake8 checking for real. I suggest we do that.